src/turnkeyml/build/export.py

```diff
@@ -279,9 +280,31 @@ def fire(self, state: build.State):
         default_warnings = warnings.showwarning
         warnings.showwarning = _warn_to_stdout
 
+        stats = fs.Stats(
+            state.cache_dir, state.config.build_name, state.evaluation_id
+        )
+
+        # Verify if the exported model matches the input torch model
+        try:
+            # The `torch.onnx.verification.find_mismatch()` takes input arguments to the
+            # model as `input_args (Tuple[Any, ...])`
+            export_verification = torch.onnx.verification.find_mismatch(
+                state.model,
+                tuple(state.inputs.values()),
+                opset_version=state.config.onnx_opset)
+            is_export_valid = not export_verification.has_mismatch()
+        except Exception: # pylint: disable=broad-except
```

Comment:
I would prefer if you could elaborate the specific expected exceptions here. Especially since there may be some distinction between "unverified" vs. a verification that actually failed. 

Failed verifications should have some specific indication like `is_export_valid="failed"`

Reply:
Added detailed comments